.github/workflows/ci.yml

```
@@ -11,7 +11,7 @@ jobs:
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        julia-version: ["1.6", "1.9", "1.10"]
+        julia-version: ["1.8", "1.9", "1.10"]
```

```suggestion
        julia-version: ["1.6", "1.9", "1.10"]
```
Wasn't the issue with Julia 1.6 resolved already?

---

I don't remember, let's ask CI ;)

---

Indeed! Thanks for noticing/reminding, and great to be back on 1.6 even.